packages/manager/src/features/Longview/LongviewDetail/DetailTabs/OverviewGraphs/DiskGraph.tsx

```diff
@@ -39,10 +40,10 @@ export const DiskGraph = (props: GraphProps) => {
   const { error, read, swap, write } = React.useMemo(
     () =>
       processDiskData(
-        pathOr({}, ['Disk'], data),
-        pathOr('kvm', ['SysInfo', 'type'], data)
+        db1disk[0].DATA.Disk,
+        db1disk[0].DATA.SysInfo.type
       ),
```

Comment:
When we remove the mock data, we can remove the ramda util too:

```suggestion
    () => processDiskData(data.Disk ?? {}, data.SysInfo?.type ?? 'kvm'),
```

Reply:
Same with the other usages of `pathOr` in this file so we can get rid of the ramda import all together.